vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1518,4 +1522,108 @@ winrt::Microsoft::ReactNative::ComponentView WindowsTextInputComponentView::Crea
   return winrt::make<WindowsTextInputComponentView>(compContext, tag, reactContext);
 }
 
+void WindowsTextInputComponentView::autoCapitalizeOnUpdateProps(const std::string &capitalizationType) noexcept {
+  /*
+    Possible values are:
+     Characters - All characters.
+     Words - First letter of each word.
+     Sentences - First letter of each sentence.
+     None - Do not autocapitalize anything.
+  */
+
+  BSTR bstr = nullptr;
+  winrt::hresult hr = m_textServices->TxGetText(&bstr);
+
+  if (SUCCEEDED(hr)) {
+    std::wstring wstrText = std::wstring(bstr, SysStringLen(bstr));
+
+    if (capitalizationType == "characters") {
+      std::transform(wstrText.begin(), wstrText.end(), wstrText.begin(), towupper);
+    } else if (capitalizationType == "words") {
+      bool nextWord = true;
```

Comment:
I'll keep it as is. The main point of doing curly brace initializations is to guarantee type safety. Since this is doing direct boolean initialization (rather than `bool foo = 1`, let's say), doing `bool foo{true}` or `bool foo = true` won't affect how the compiler processes it.